ui/app/mainui/activitycenter/views/ActivityNotificationMessage.qml

```diff
@@ -0,0 +1,68 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+import shared 1.0
+import utils 1.0
+import shared.views.chat 1.0
+
+ActivityNotificationBase {
+    id: root
+
+    property var messageContextMenu
+
+    signal activityCenterClose()
+
+    height: Math.max(60, notificationMessage.height)
+
+    MessageView {
+        id: notificationMessage
+        anchors.top: parent.top
+        anchors.left: parent.left
+        anchors.right: parent.right
+        z: -1
+
+        rootStore: root.store
```

Comment:
There is no `root.store` property, how does this work?

Reply:
ActivityNotificationMessage is derived from ActivityNotificationBase which has store and notification

Reply:
Ah! :)